examples/predict.py

```diff
@@ -0,0 +1,28 @@
+import t4
+import os
+import tqdm
+import json
+import pandas as pd
+
+from fnet.cli.init import save_default_train_options
```

Comment:
First, why is this import needed? Also, my understanding is that this is supposed to be an example of the `fnet` Python API, yet this file does not use the Python API. Wouldn't a bash script be clearer since you use the command-line interface here? Also, keep in mind that the user can run `fnet init` on the command line to see example scripts of how the `fnet` module can be used.